app/src/main/java/com/arqoders/hotelapp/data/server/TheHotelDb.kt

```diff
@@ -0,0 +1,21 @@
+package com.arqoders.hotelapp.data.server
+
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+class TheHotelDb(baseUrl: String) {
```

Comment:
Cambiar el nombre del archivo y la clase

Reply:
Renamed to RetrofitClientInstance